app/views/home/partials/_nav.html.erb

```diff
@@ -0,0 +1,28 @@
+<section>
+  <div>
+    <div>
+      <%# SVG %>
+    </div>
+    
+    <ul>
+      <li>
+        <a href="#products" data-controller="scroll-to">PRODUCTS</a>
+      </li>
+      <li>
+        <a href="#pricing" data-controller="scroll-to">PRICING</a>
+      </li>
+      <li>
+        <a href="about-us" data-controller="scroll-to">ABOUT US</a>
+      </li>
+    </ul>
+    
+    <div>
+      <%= link_to "Login", new_user_session_path %>
+      <%= link_to "Register", new_user_registration_path %>
+    </div>
+
+    <div>
+      <%# SVG %>
+    </div>
+  </div>
```

Comment:
Question, why do you need to group the elements in a div, if you are grouping them in a section?

Reply:
We are using SASS for the development of styles, so mainly when I work with sass I add a main class to wrap the entire element and the second div would be to make the container where the use of flexbox would be reflected.

example: 

![image](https://user-images.githubusercontent.com/50375089/143970527-d548ca6a-ad6f-4c65-977f-ab30a3ebfda3.png)